frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortletUtils.java

```diff
@@ -54,32 +56,43 @@ private ComponentPortletUtils() {
 
     public static void updateReleaseFromRequest(PortletRequest request, Release release) {
         for (Release._Fields field : Release._Fields.values()) {
-            switch (field) {
-                case REPOSITORY:
-                    release.setFieldValue(field, getRepositoryFromRequest(request));
-                    break;
-                case CLEARING_INFORMATION:
-                    release.setFieldValue(field, getClearingInformationFromRequest(request));
-                    break;
-                case COTS_DETAILS:
-                    release.setFieldValue(field, getCOTSDetailsFromRequest(request));
-                    break;
-                case VENDOR_ID:
-                    release.unsetVendor();
-                    setFieldValue(request, release, field);
-                    break;
-                case ATTACHMENTS:
-                    if (!release.isSetAttachments()) release.setAttachments(new HashSet<Attachment>());
-                    PortletUtils.updateAttachmentsFromRequest(request, release.getAttachments());
-                    break;
-                case RELEASE_ID_TO_RELATIONSHIP:
-                    if (!release.isSetReleaseIdToRelationship()) release.setReleaseIdToRelationship(new HashMap<String, ReleaseRelationship>());
-                    updateLinkedReleaesFromRequest(request, release.releaseIdToRelationship);
-                    break;
-                default:
-                    setFieldValue(request, release, field);
+            if (field != Release._Fields.ATTACHMENTS) {
```

Comment:
Couldn't that also be done with a
```Java
[...]
            case Release._Fields.ATTACHMENTS:
                break;
[...]
```
This would avoid one level of nesting.